portal/static/js/admin.js

```diff
@@ -359,5 +334,67 @@ AdminTool.prototype.getInstrumentList = function() {
   };
 };
 
+AdminTool.prototype.handleDownloadModal = function() {
+
+    var self = this;
+     /* 
+      *populate instruments list based on user's parent org
+      */
+    $("#dataDownloadModal").on('shown.bs.modal', function () {
+          var parentOrgList = AT.getUserTopLevelParentOrgs(AT.getUserOrgs());
+          console.dir(parentOrgList)
```

Comment:
I'm referring to the `console.dir()` call - I assume that was a debugging statement left behind?  IIRC some <cough> browsers don't do well with lots of console logging.